packages/firebase_ml_vision/example/android/app/src/main/java/io/flutter/plugins/firebasemlvisionexample/MainActivity.java

```diff
@@ -1,13 +1,23 @@
 package io.flutter.plugins.firebasemlvisionexample;
 
-import android.os.Bundle;
-import io.flutter.app.FlutterActivity;
-import io.flutter.plugins.GeneratedPluginRegistrant;
+import io.flutter.embedding.android.FlutterActivity;
+import io.flutter.embedding.engine.FlutterEngine;
+import io.flutter.embedding.engine.plugins.shim.ShimPluginRegistry;
+import io.flutter.plugins.firebasemlvision.FirebaseMlVisionPlugin;
+import io.flutter.plugins.imagepicker.ImagePickerPlugin;
+import io.flutter.plugins.pathprovider.PathProviderPlugin;
 
 public class MainActivity extends FlutterActivity {
   @Override
-  protected void onCreate(Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
-    GeneratedPluginRegistrant.registerWith(this);
+  public void configureFlutterEngine(FlutterEngine flutterEngine) {
+    flutterEngine.getPlugins().add(new FirebaseMlVisionPlugin());
+
+    final ShimPluginRegistry shimPluginRegistry = new ShimPluginRegistry(flutterEngine);
+    PathProviderPlugin.registerWith(
+        shimPluginRegistry.registrarFor("io.flutter.plugins.pathprovider.PathProviderPlugin"));
+    ImagePickerPlugin.registerWith(
+        shimPluginRegistry.registrarFor("io.flutter.plugins.imagepicker.VideoPlayerPlugin"));
+    //    CameraPlugin.registerWith(
```

Comment:
This is commented out because the `CameraPlugin` requires an v1 embedding `FlutterView`. This prevented the example app from running, but shouldn't affect the plugin dependencies.